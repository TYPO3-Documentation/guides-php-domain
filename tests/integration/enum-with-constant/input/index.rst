===================
PHP Enum with Const
===================

..  php:enum:: MyVendor\MyExtension\SomeEnum

    some stuff


    ..  php:const:: PI

        represents the ratio of a circle's circumference to its diameter

