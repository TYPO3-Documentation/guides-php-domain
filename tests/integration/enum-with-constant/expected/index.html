<!-- content start -->
    <div class="section" id="php-enum-with-const">
            <h1>PHP Enum with Const</h1>
            <dl class="php enum">
    <dt class="sig sig-object php" id="myvendor-myextension-someenum">
        <em class="property"><span class="pre">enum</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\MyVendor\MyExtension\</span></span>
<span class="sig-name descname"><span class="pre">SomeEnum</span></span> <a class="headerlink" href="#myvendor-myextension-someenum" title="Permalink to this definition">¶</a>
    </dt>
    <dd>

    <p>some stuff</p>
<dl class="php const">
    <dt class="sig sig-object php" id="myvendor-myextension-someenum-pi">
        <em class="property"><span class="pre">const</span></em>
        <span class="sig-name descname"><span class="pre">PI</span></span>
         <a class="headerlink" href="#myvendor-myextension-someenum-pi" title="Permalink to this definition">¶</a>
    </dt>
    <dd>
    <p>represents the ratio of a circle&#039;s circumference to its diameter</p>
</dd>
</dl>
    </dd>
</dl>
    </div>
<!-- content end -->
